src/index.html

```diff
@@ -11,7 +11,174 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      onsubmit="onSubmit()"
+    >
+      <fieldset class="form_fieldSet">
+        <legend>Personal information</legend>
+        <div class="form_field">
+          <label for="Surname">Surname:</label>
+          <input
+            type="text"
+            name="Surname"
+            id="Surname"
+          ></input>
+        </div>
+        <div class="form_field">
+          <label for="Name">Name:</label>
+          <input
+            type="text"
+            name="Name"
+            id="Name"
+          ></input>
+        </div>
+        <div class="form_field">
+          <label for="Age">How old are You?</label>
+          <input
+            type="number"
+            name="Age"
+            min="12"
+            max="100"
+            value="12"
+            id="Age"
+          ></input>
+        </div>
+        <div class="form_field">
+          <label for="BirthDate">Full date of birth:</label>
```

Comment:
If you use camelCase for id's, make sure you use it properly
```suggestion
          <label for="birthDate">Full date of birth:</label>
```